staging/src/k8s.io/apiextensions-apiserver/pkg/registry/customresource/etcd.go

```diff
@@ -52,3 +53,11 @@ func NewREST(resource schema.GroupResource, listKind schema.GroupVersionKind, st
 	}
 	return &REST{store}
 }
+
+// Implement CategoriesProvider
+var _ rest.CategoriesProvider = &REST{}
+
+// Categories implements the CategoriesProvider interface. Returns a list of categories a resource is part of.
+func (r *REST) Categories() []string {
+	return []string{"all"}
```

Comment:
Not sure we really want that. The resources in `all` is manually curated because we don't want to see certain resources in there that are internal and not relevant for a user.

Reply:
Yeah, we don't really want this here.  I could see allowing a user to specify their categories.

Reply:
> I could see allowing a user to specify their categories.

Maybe as another field in the CRD spec (?)

Reply:
Exactly